Sources/Build/Command.link(ClangModule).swift

```diff
@@ -28,7 +28,7 @@ extension Command {
             let buildMeta = ClangModuleBuildMetadata(module: module, prefix: prefix, otherArgs: [])
             objects += buildMeta.objects
             inputs += buildMeta.inputs
-            linkFlags += buildMeta.linkDependenciesFlags
+            linkFlags += buildMeta.linkDependenciesFlags + module.languageLinkArgs
```

Comment:
I don't think it is this simple, we need to aggregate across all modules in the transitive closure -- an executable just written in C might depend on a C++ library.